kore/src/Kore/Log/ErrorRewriteRuleId.hs

```diff
@@ -0,0 +1,42 @@
+{- |
+Copyright   : (c) Runtime Verification, 2020
+License     : NCSA
+
+-}
+
+module Kore.Log.ErrorRewriteRuleId
+    ( ErrorRewriteRuleId
+    , errorRewriteRuleId
+    ) where
+
+import Prelude.Kore
+
+import Data.Text.Prettyprint.Doc as Pretty
+
+import Kore.Step.RulePattern
+    ( RewriteRule
+    , getRewriteRule
+    )
+import Kore.Syntax.Variable
+    ( Variable
+    )
+
+import Log
+
+newtype ErrorRewriteRuleId =
```

Comment:
I think the name of this error is slightly misleading because it suggests that the rewrite rule has an "identity" form, but actually the right-hand side is under `\next`. I would prefer if we called this `ErrorRewriteLoop`; the name suggests what the problem is.